### PR TITLE
PricelessPositionManager.sol natspec

### PR DESCRIPTION
**Pull request by chrismaree**
This PR adds commenting and formatting changes to the `PricelessPositionManager.sol` such that it conforms to the solidity Natspec and styling guide.

### CODE REVIEW

File: core/contracts/financial-templates/implementation/PricelessPositionManager.sol [hunk 1]
```diff
@@ -363,11 +377,29 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
         emit SettleExpiredPosition(msg.sender, totalRedeemableCollateral.rawValue, tokensToRedeem.rawValue);
     }
 
+    /****************************************
+     *        GLOBAL STATE FUNCTIONS        *
+     ****************************************/
+
+    /**
+     * @notice After expiration of the contract the DVM is asked what the prevailing price at the
+     * time of expiration. Final fees are paid. Once price is resolved token holders can withdraw.
```
Comment (mrice32) [inline core/contracts/financial-templates/implementation/PricelessPositionManager.sol]: nit: maybe reword this comment? After reading, it's a little difficult to understand which part of this the `expire` method does vs what happens after.

File: core/contracts/financial-templates/implementation/PricelessPositionManager.sol [hunk 2]
```diff
@@ -395,14 +427,18 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
 
     /**
      * @notice Accessor method for a sponsor's collateral.
-     * @dev This is necessary because the struct returned by the positions() method shows rawCollateral, which isn't a
-     * user-readable value.
+     * @dev This is necessary because the struct returned by the positions() method shows
+     * rawCollateral, which isn't a user-readable value.
+     * @param sponsor address who's collateral amount is retreived for.
```
Comment (mrice32) [inline core/contracts/financial-templates/implementation/PricelessPositionManager.sol]:
```suggestion
     * @param sponsor address whose collateral amount is retrieved.
```

File: core/contracts/financial-templates/implementation/PricelessPositionManager.sol [hunk 3]
```diff
@@ -395,14 +427,18 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
...
      */
     function getCollateral(address sponsor) external view returns (FixedPoint.Unsigned memory) {
         // Note: do a direct access to avoid the validity check.
         return _getCollateral(positions[sponsor].rawCollateral);
     }
 
+    /**
+     * @notice Accessor method for the total collateral stored within the PricelessPositionmanager.
```
Comment (mrice32) [inline core/contracts/financial-templates/implementation/PricelessPositionManager.sol]:
```suggestion
     * @notice Accessor method for the total collateral stored within the PricelessPositionManager.
```

File: core/contracts/financial-templates/implementation/PricelessPositionManager.sol [hunk 4]
```diff
@@ -17,8 +17,8 @@ import "./FeePayer.sol";
 
 /**
  * @title Financial contract with priceless position management.
- * @notice Handles positions for multiple sponsors in an optimistic (i.e., priceless) way without relying on a price feed.
- * On construction, deploys a new ERC20 that this contract manages that is the synthetic token.
+ * @notice Handles positions for multiple sponsors in an optimistic (i.e., priceless) way without
+ * relying on a price feed. On construction, deploys a synthetic ERC20 managed by this contract.
```
[the remark this answers is not in the document]
Comment (chrismaree) [reply core/contracts/financial-templates/implementation/PricelessPositionManager.sol]: ye, good point. re-worded it

File: core/contracts/financial-templates/implementation/PricelessPositionManager.sol (hunk 1 above)
Comment (chrismaree) [reply core/contracts/financial-templates/implementation/PricelessPositionManager.sol]: ye, sorry about that. have fixed and pushed
